web/src/table/SignupTable.js

```diff
@@ -291,6 +312,33 @@ class SignupTable extends React.Component {
           );
         },
       },
+      {
+        title: i18next.t("signup:Custom Item Field"),
+        dataIndex: "customItemField",
+        key: "customItemField",
+        render: (text, record, index) => {
+          const initialValue = text || "";
+          const isMultiple = record.rule === "Multiple Choices";
+          const isSingle = record.rule === "Single Choice";
```

Comment:
is seems that we can add input or check... in later version, you can check this by use 
```js
["Multiple Choices", "Single Choice"].includes(record.rule)
```